docs/content/users/install/performance.md

```diff
@@ -163,8 +171,7 @@ Mutagen is enabled by default on Mac and traditional Windows, and it can be disa
 
     #### Advanced Mutagen Troubleshooting
 
-    Most people get all the information they need about mutagen by running `ddev mutagen monitor` to see the results. However, Mutagen has full logging. You can run it with `ddev mutagen logs`.
-
+    Most people get all the information they need about mutagen by watching `ddev mutagen monitor` in another terminal window to see the results. However, Mutagen has full logging. You can run it with `ddev mutagen logs`.
```

Comment:
I’d capitalize Mutagen and reword this so it doesn’t mention “most people” and leave you feeling like a bizarre and uncommon creature if you’re not getting the information you need—get to the point and say what it’s for:

```suggestion
    You can observe what Mutagen is doing by watching `ddev mutagen monitor` in another terminal window to see the results. However, Mutagen has full logging. You can run it with `ddev mutagen logs`.
```

Reply:
I’m not sure why “Mutagen has full logging” is prefaced with “However,” but I’m trying to review quickly here. (Does that mean `ddev mutagen monitor` is more convenient, or more abbreviated? Why would I use `ddev mutagen monitor` vs. `ddev mutagen logs`?)